build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -12,10 +12,39 @@
     <!-- xUnit.net WPF Runner -->
     <XUnitWpfPath>$(NuGetPackageRoot)\xunit.runner.wpf\$(xunitrunnerwpfVersion)\tools\xunit.runner.wpf.exe</XUnitWpfPath>
     <XUnitWpfArguments>"$(OutDir)\$(AssemblyName).dll"</XUnitWpfArguments>
-  </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="xunit.runner.wpf" Version="$(xunitrunnerwpfVersion)" PrivateAssets="all" />
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <ForceGenerationOfBindingRedirects>true</ForceGenerationOfBindingRedirects>
+
+    <RuntimeIdentifier Condition="'$(TargetFramework)' == 'net461' AND '$(RuntimeIdentifier)' == '' AND '$(OS)' == 'Windows_NT'">win7</RuntimeIdentifier>
+  </PropertyGroup>
+    
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+    <Content Include="$(NetStandard20RefPath)\netstandard.dll">
+      <Link>ref/%(FileName)%(Extension)</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Visible>false</Visible>
+    </Content>
+    <Content Include="$(NetStandard20RefPath)\mscorlib.dll">
+      <Link>ref/%(FileName)%(Extension)</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Visible>false</Visible>
+    </Content>
+    <Content Include="$(NetStandard20RefPath)\System.Runtime.dll">
+      <Link>ref/%(FileName)%(Extension)</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Visible>false</Visible>
+    </Content>
+    <Content Include="$(NetStandard20RefPath)\System.Dynamic.Runtime.dll">
+      <Link>ref/%(FileName)%(Extension)</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <Visible>false</Visible>
+    </Content>
+    <Content Include="$(MicrosoftCSharpRefPath)\Microsoft.CSharp.dll">
```

Comment:
@nguerrera I found why the build copies the ref for Microsoft.CSharp.dll from nuget packages.

`build/Targets/Settings.props:    <MicrosoftCSharpRefPath>$(NuGetPackageRoot)/microsoft.csharp/$(MicrosoftCSharpVersion)/ref/netstandard1.0/</MicrosoftCSharpRefPath>`

Is the fallback folder for packages a new behavior of the 15.5/2.1 SDK?

Reply:
The name "ref" here is probably a bit misleading. This has nothing to do with our actual build semantics. Rather it the directory that our core unit test function (compile, verify and run) look for reference assemblies in. We should probably rename this to "UnitTestRef" to make it much more apparent what the function of this folder is.

Reply:
The fallback folder was in 2.0 as well. We'll have to dig in to understand why it wasn't populated/used before this change. There are supported ways to opt out (if need be), but I wouldn't expect any difference here between 2.0 and 2.1/15.5 so something unexpected is going on.